docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to Stake on EON
+---
+
+# How to Stake on EON
```

Comment:
This whole section is missing information on:
1. generating keys in an alternate way using the bootstrapping tool
2. importing existing keys using the node config file if running the node natively, or the docker .env if using compose-evm-simplified
3. the behavior of secrets import from config file on first run only
4. how secrets generated in the local node are tied to the seed in the config file
5. configuring the node through the config to automatically start forging
6. listing existing forging stakes
7. withdrawing forging stakes

Reply:
Can be addressed at a later time.